File: Java2/src/Array_2D/Q_3.java
package Array_2D;

import java.util.Scanner;

public class Q_3 {
    public static void main(String[] args) {

        Scanner sc =new Scanner(System.in);
        System.out.println("Enter your row size :");
        int row=sc.nextInt();
//        int colum=sc.nextInt();
        int arr[][]=new int[row][];
        for(int i=0;i<row;i++){
            System.out.println("Enter your colum size for  row index : "+i);
             arr[i]=new int[sc.nextInt()];
        }
        System.out.println("Enter your array Element :");
        for(int i=0;i<row;i++){
            for(int j=0;j<arr[i].length;j++){
                arr[i][j]= sc.nextInt();
            }
        }
        for(int i=0;i<row;i++){
            for(int j=0;j<arr[i].length;j++){
                System.out.print(arr[i][j]);
            }
            System.out.println();
        }

    }
}

File: one_d_array/Assingment/Q_3.c
#include<stdio.h>
int main(){
    int size;
    printf("Enter size :");
    scanf("%d",&size);
    int arr[size];
    printf("Enter Element :");
    int sum=0;
    for(int i=0;i<size;i++){
        scanf("%d",&arr[i]);
        sum=sum+arr[i];
    }
    printf("%d",sum);
return 0;
}